firebase-firestore/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - [feature] Added a `@DocumentId` annotation which can be used on a
   `DocumentReference` or `String` property in a POJO to indicate that the SDK
   should automatically populate it with the document's ID.
+- [changed] Failed transactions now throw the failure from the last attempt,
+  instead of `ABORTED.` 
```

Comment:
"throw" and "failure" don't go together that well. Usually you throw exceptions. However, to the end user we don't really throw anyway, since what they see is a `Task`. Tasks fail with exceptions.

Also, "ABORTED" here is _code_ the associated with the exception, not the thing we actually return.

How about:
> Failed transactions now fail with the exception from the last attempt instead of an always failing with an exception with code `ABORTED`.

Reply:
Ah, that sounds so much less confusing. I'll update the iOS to reflect this as well.